tests/test_settings_effect.py

```diff
@@ -47,9 +47,13 @@ def test_backup_config(
         return
     # Check selected duplicity image
     if "s3" in backup_dst:
-        assert prod["services"]["backup"]["image"] == "tecnativa/duplicity:postgres-s3"
+        assert prod["services"]["backup"]["image"].startswith(
+            "ghcr.io/tecnativa/docker-duplicity-postgres-s3:"
+        )
     else:
-        assert prod["services"]["backup"]["image"] == "tecnativa/duplicity:postgres"
+        assert prod["services"]["backup"]["image"].startswith(
+            "ghcr.io/tecnativa/docker-duplicity-postgres:"
+        )
```

Comment:
@Yajo WDYT about leaving this test like this? It would avoid having to change the test each time we change the docker image tag, while still checking the name is correct...

Reply:
Can't you just give an answer to the copy process and then assert that the answer is correctly used here?